association.go

```diff
@@ -1963,8 +1965,8 @@ func (a *Association) handleChunk(p *packet, c chunk) error {
 	var err error
 
 	if _, err = c.check(); err != nil {
-		return errors.Wrap(err, "failed validating chunk")
-		// TODO: Create ABORT
```

Comment:
I didn't find anything in the RFC, but it seems like we should tolerate anything from the other side (besides ABORT chunks)

Reply:
There are some cases mentioned in RFC 4960 which require sending ABORT  to close the association. (grep by `should abort` or `must abort` in the RFC).
Chunks that can return an error from check() currently are chunk_init.go and chunk_init_ack.go as far as I can tell.
We've been ignoring those errors so the logic hasn't be proven. I'd support you change (just log and ignore) for now to reduce regression. We could revisit this later when we finally support "sending ABORT".

Reply:
Ticketed! https://github.com/pion/sctp/issues/103